### ISSUE TITLE
[bug]: bitcoind sometimes hard restarts, losing sync progress, even after synced 100%

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/bitcoind-wrapper#42]
Sometimes bitcoind just dies. This results in lost sync progress. I recently had this happen and it started resyncing 200 blocks back.

It is not being OOM killed, since there is no record of this in dmesg.

No idea what else it could be

**Comment by ProofOfKeags** [comment]
Are we able to find that in docker logs?

**Comment by chrisguida** [comment]
Blocked by https://github.com/Start9Labs/embassy-os/issues/1155

**Comment by ProofOfKeags** [comment]
why does this depend on Start9Labs/embassy-os#1138?

**Comment by chrisguida** [comment]
It might not. I thought that was the "get resources under control" ticket. If it's not relevant we can remove it.

**Comment by ProofOfKeags** [comment]
I'd rather not designate it as BLOCKED on 1138. Having the relation link is fine.

**Comment by chrisguida** [comment]
I had blocked on there because Aiden suggested that we wait until the system-level resource control is finished before revisiting this.

**Comment by chrisguida** [comment]
The case above is not an autoreindex. Notice that only a couple hundred blocks were dropped. This is more likely to do with an ungraceful shutdown. The question is why the ungraceful shutdown occurred. The autoreindex/ntp bug is a separate issue

**Comment by ProofOfKeags** [comment]
Does that other bug have a ticket or do I need to track it?

**Comment by chrisguida** [comment]
Just created one now: https://github.com/Start9Labs/embassy-os/issues/1193
